sympy/stats/crv_types.py

```diff
@@ -397,6 +399,97 @@ def Beta(name, alpha, beta):
 
     return rv(name, BetaDistribution, (alpha, beta))
 
+#-------------------------------------------------------------------------------
+# Noncentral Beta distribution ------------------------------------------------------------
+
+
+class BetaNoncentralDistribution(SingleContinuousDistribution):
+    _argnames = ('alpha', 'beta', 'lamda')
+
+    set = Interval(0, 1)
+
+    @staticmethod
+    def check(alpha, beta, lamda):
+        _value_check(alpha > 0, "Shape parameter Alpha must be positive.")
```

Comment:
Symbolic parameters should be allowed (here and below).
```suggestion
        _value_check((alpha > 0) != False, "Shape parameter Alpha must be positive.")
```

Reply:
@sidhantnagpal I have the following doubts:
1. If you set `alpha = Symbol('alpha', positive=True)` then `alpha > 0` evaluates to true and so I believe that `_value_check(alpha > 0, ...)` should work for Symbolic parameters.
2. I am facing a lot of problem regarding docstring. Since I have long outputs and can't use `\`, I am not able to pass docstring test. Please help.

Reply:
> should work for Symbolic parameters

It should also work when `is_positive` gives `None`. That is when `alpha = Symbol('alpha')`.

Reply:
> Since I have long outputs and can't use `\`

This will helpful: https://docs.python.org/3/library/doctest.html#how-are-docstring-examples-recognized.

Reply:
> Since I have long outputs and can't use `\`

This will be helpful: https://docs.python.org/3/library/doctest.html#how-are-docstring-examples-recognized

Reply:
I have added the following test in `BetaNoncentral`
```
a = Symbol('a', positive=False)
raises(ValueError, lambda: BetaNoncentral('x', a, b, c))

a = Symbol('a', positive=True)
b = Symbol('b', positive=False)
raises(ValueError, lambda: BetaNoncentral('x', a, b, c))

a = Symbol('a', positive=True)
b = Symbol('b', positive=True)
c = Symbol('c', nonnegative=False)
raises(ValueError, lambda: BetaNoncentral('x', a, b, c))
```
If I allow for Symbolic parameter then they would be meaningless. What do you suggest?

Reply:
It seems that passing `alpha > 0` in `_value_check` is able to handle both cases when `alpha` has `positive` set to `True` or `None` and raises exception only when it is `False`. So either of the checks could be kept. Also see https://github.com/sympy/sympy/pull/16576#discussion_r282833876. The key point is that exception should be raised only if the symbols violate the expected conditions (and not when the assumption of the symbol can not be determined).

Reply:
@sidhantnagpal thank you for the above note. I have verified that `_value_check` indeed handle both cases. I have also added a test in `sympy/stats/tests/test_continuous_rv` for symbolic parameters when assumption on the symbols is specifically given.